docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -415,18 +415,26 @@ spec:
 
 .Verification
 
-* Verify that MariaDB and RabbitMQ are running for all defined cells:
+* Verify that the Galera and RabbitMQ status is `Running` for all defined cells:
 +
+[source,yaml]
 ----
 $ RENAMED_CELLS="cell1 cell2 cell3"
-$ oc get pod openstack-galera-0 -o jsonpath='{.status.phase}{"\n"}' | grep Running
-$ oc get pod rabbitmq-server-0 -o jsonpath='{.status.phase}{"\n"}' | grep Running
+$ oc get pod openstack-galera-0 -o jsonpath='{.status.phase}{"\n"}'
+$ oc get pod rabbitmq-server-0 -o jsonpath='{.status.phase}{"\n"}'
 $ for CELL in $(echo $RENAMED_CELLS); do
-    oc get pod openstack-$CELL-galera-0 -o jsonpath='{.status.phase}{"\n"}' | grep Running
-    oc get pod rabbitmq-$CELL-server-0 -o jsonpath='{.status.phase}{"\n"}' | grep Running
-done
+>     oc get pod openstack-$CELL-galera-0 -o jsonpath='{.status.phase}{"\n"}'
+>     oc get pod rabbitmq-$CELL-server-0 -o jsonpath='{.status.phase}{"\n"}'
+> done
```

Comment:
I think it may make sense to include a sample of expected output here.

Reply:
The expected output is the word `Running` per cell, i tried to hint at that in the introductory sentence for the code block. It could be made more clear a the cost of being more verbose, but i'm not sure if that's worth respinning the CI.

Reply:
We can be more verbose in documentation and have grep in CI at the same time.
Here is example output:
`$ oc get pods -n openstack rabbitmq-server-0  
NAME                READY   STATUS    RESTARTS   AGE
rabbitmq-server-0   1/1     Running   0          51m`

and for galera:
` oc get pods -n openstack openstack-galera-0
NAME                 READY   STATUS    RESTARTS   AGE
openstack-galera-0   1/1     Running   0          53m`